Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM python:3.6
 
+RUN pip install -U pipenv==8.2.7
```

Comment:
I think we should remove -U as it contradicts the pinned version? 
Thats probably why codacy is moaning.